src/sentry/static/sentry/app/components/search/index.jsx

```diff
@@ -92,6 +92,11 @@ class Search extends React.Component {
 
     if (!to) return;
 
+    if (to.startsWith('http')) {
+      window.open(to, '_blank');
```

Comment:
Let's make sure opener is null, see https://mathiasbynens.github.io/rel-noopener/

Reply:
Fixed.